src/content/en/_shared/feedback.html

```diff
@@ -3,13 +3,14 @@
   usage.
 {% endcomment %}
 {% framebox width="auto" height="auto" enable_widgets="true" %}
-{% setvar url %}https://github.com/google/webfundamentals/issues/new?title=[{{ category }} / {{ label }}] {% endsetvar %}
```

Comment:
Instead of removing this, and having to define in in all use cases, why not use something like this:
```
{% if url %}
{% setvar feedbackURL %}{{url}}{% endsetvar %}
{% else %}
{% setvar feedbackURL %}https://github.com/google/webfundamentals/issues/new?title=[{{ category }} / {{ label }}]{% endsetvar %}
{% endif %}

{{feedbackURL}}
```

If the URL is defined, it uses that, but otherwise it uses the fallback URL.